js/main.js

```diff
@@ -2,6 +2,12 @@ import Books from './books.js';
 
 const NewBooks = new Books();
 
+function assignDate() {
+  const DateTime = luxon.DateTime.now(); // eslint-disable-line no-undef
+  const Date = `${DateTime.year} - ${DateTime.month} - ${DateTime.day}`;
+  document.getElementById('date').innerHTML = Date;
+}
+
```

Comment:
- [ ] Kindly ensure arrow functions are used instead of the function keyword. Javascript's ES6 brings new syntax and awesome features to make your code more modern and readable. It allows you to write less code and do more. ES6 introduces us to many great features like arrow functions. The arrow function is really awesome and makes your code more readable, more structured, and look modern. 
- [ ] In the wireframe given, the time is also displayed alongside the date.Kindly add the time as well

Reply:
Thank you for reviewing our project! We did all the requested changes that you asked for. Have a nice day!